src/com/android/settings/wifi/WiFiTimeoutPreferenceController.java

```diff
@@ -0,0 +1,46 @@
+package com.android.settings.wifi;
+
+import android.content.Context;
+import android.provider.Settings;
+
+import androidx.preference.Preference;
+
+import com.android.settings.R;
+import com.android.settings.core.BasePreferenceController;
+
+public class WiFiTimeoutPreferenceController extends BasePreferenceController {
+
+    private final Context mContext;
```

Comment:
`protected final Context mContext;` is already declared in `AbstractPreferenceController` (super-class of `BasePreferenceController`), which means it's accessible here too, no need to declare it for the second time.

Reply:
okay, thanks.